tests/plugins/plugin.text_selection.test.js

```diff
@@ -0,0 +1,102 @@
+import sinon from 'sinon';
+import '../../BookReader/jquery-1.10.1.js';
+import '../../BookReader/jquery-ui-1.12.0.min.js';
+import '../../BookReader/jquery.browser.min.js';
+import '../../BookReader/dragscrollable-br.js';
+import '../../BookReader/jquery.colorbox-min.js';
+
+import '../../src/js/BookReader.js';
+import BookreaderWithTextSelection from '../../src/js/plugins/plugin.text_selection.js';
+import { PageModel, BookModel } from '../../src/js/BookReader/BookModel.js';
+
+/** @type {BookReader} */
+const FAKE_XML = `<OBJECT data="file://localhost//tmp/derive/goodytwoshoes00newyiala//goodytwoshoes00newyiala.djvu" height="3192" type="image/x.djvu" usemap="goodytwoshoes00newyiala_0001.djvu" width="2454">
```

Comment:
Add a comment with the link to the djvu where you copied this from

Reply:
done